### PR TITLE
FSx Tutorial

### PR DESCRIPTION
**Pull request by sean-smith**
Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/tutorials/code_samples/fsx/fsx_postinstall.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Install lustre drivers
+wget https://downloads.whamcloud.com/public/lustre/lustre-2.10.5/el7.5.1804/client/RPMS/x86_64/kmod-lustre-client-2.10.5-1.el7.x86_64.rpm
+wget https://downloads.whamcloud.com/public/lustre/lustre-2.10.5/el7.5.1804/client/RPMS/x86_64/lustre-client-2.10.5-1.el7.x86_64.rpm
+
+yum localinstall -y *lustre-client-2.10.5*.rpm
+
+# Mount the file system
+
+mkdir -p $mnt
```
Comment (mauri-melato) [inline docs/tutorials/code_samples/fsx/fsx_postinstall.sh]: "${mnt}"

File: docs/tutorials/code_samples/fsx/fsx_postinstall.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+mount -t lustre ${fsid}.fsx.${cfn_region}.amazonaws.com@tcp:${mnt} ${mnt}
```
Comment (mauri-melato) [inline docs/tutorials/code_samples/fsx/fsx_postinstall.sh]: double quotes around the two arguments

File: docs/tutorials/code_samples/fsx/fsx_postinstall.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+. /etc/parallelcluster/cfnconfig
+
+fsid=$2
+mnt=$3
+
+if [ "x$fsid" == "x" ]; then
+  echo "ERROR: You must provide the FSx File System Id i.e. fs-00079dd40d69349ce"
+  exit 1
+fi
+if [ "x$mnt" == "x" ]; then
```
Comment (mauri-melato) [inline docs/tutorials/code_samples/fsx/fsx_postinstall.sh]: {}

File: docs/tutorials/code_samples/fsx/fsx_postinstall.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+. /etc/parallelcluster/cfnconfig
+
+fsid=$2
+mnt=$3
+
+if [ "x$fsid" == "x" ]; then
```
Comment (mauri-melato) [inline docs/tutorials/code_samples/fsx/fsx_postinstall.sh]: curly braces around var name

File: docs/tutorials/code_samples/fsx/fsx_postinstall.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+wget https://downloads.whamcloud.com/public/lustre/lustre-2.10.5/el7.5.1804/client/RPMS/x86_64/kmod-lustre-client-2.10.5-1.el7.x86_64.rpm
+wget https://downloads.whamcloud.com/public/lustre/lustre-2.10.5/el7.5.1804/client/RPMS/x86_64/lustre-client-2.10.5-1.el7.x86_64.rpm
+
+yum localinstall -y *lustre-client-2.10.5*.rpm
+
+# Mount the file system
+
+mkdir -p ${mnt}
+
+mount -t lustre "${fsid}.fsx.${cfn_region}.amazonaws.com@tcp:${mnt}" "${mnt}"
```
Comment (rajachan) [inline docs/tutorials/code_samples/fsx/fsx_postinstall.sh]: This is incorrect.

```
"${fsid}.fsx.${cfn_region}.amazonaws.com@tcp:${mnt}" "${mnt}" 
```

should be 

```
"${fsid}.fsx.${cfn_region}.amazonaws.com@tcp:/fsx" "${mnt}" 
```

The mount target is /fsx statically, regardless of the client mountpoint.

**Pull request closed without merging**